### DISCUSSION TITLE
Should `deck install` honor current context?

### DISCUSSION THREAD

**Discussion by schwobaseggl** [unikubehq/cli#86] [Ideas]
Working within the context of a certain deck `A` (set with `deck use A`), it is often necessary to install one or more other decks (`B`, `C`, ...) for apps in deck `A` to work.

In that situation, `deck install` does not provide the dropdown with available decks, but honours the set context and reinstalls deck `A`. IMO, this is counter-intuitive or at least a bit of a hazzle, because one does not want to leave/change the deck context, and in order to install deck `B` without doing so, one has to know the deck's name by heart and explicitly call `deck install B`.

```bash
deck install A
deck use A
...
deck install  // expect dropdown to select deck B, but A is installed again
```

I don't think changing this comes at a great cost. Reselecting `A` is easy and my assumption would be that the usual workflow is `install` first, then `use`, with subsequent calls to `install` likely intended to install *other* decks.

Any thoughts?

**Comment by Schille** [comment]
+1
